01week/datatypes.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+//Write a JavaScript program to display the current day and time.
+//function that receives time and date and displays upon request
+var d = new Date();
+var n = d.toLocaleString();
+  console.log(n);
+
+  // Write a JavaScript program to convert a number to a string.
+//assign variable, write function that receives a number and converts it to a string, using num.toString();
+num = 8;
```

Comment:
Wrap this in a function. "Write a program" === make a function. Also this wouldn't convert the number to a string.